draft-kazuho-httpbis-priority.md

```diff
@@ -263,6 +263,12 @@ of each image that it provides, so that an intermediary could prioritize
 certain images above others.  Or, a graphical user-agent could send a `visible`
 attribute to indicate if the resource being requested is within the viewport.
 
+Also, to improve the prioritization logic, a server can use signals other than
+that carried by the Priority header field. For example, a server might
```

Comment:
I want to check that my reading of this paragraph is what was intended. The preeceeding paragraphs talks about "in-band" signals in the form of Priority paramaters. Is this paragraph describing "out-of-band" signals? Do you think this rewording might be clearer?

_A server can combine the hints provided in the Priority header field with other information in order to improve the prioritization of responses. For example, a server that receives requests for a font {{?RFC8081}} and images with the same urgency might allocated more resources to the font, so that a visual client can render textual information at an early moment._

Reply:
That's what I meant and your sentence is better.

I've applied it with one change: I've replaced "allocate more resources" to "give higher precedence", as the former might imply that the bandwidth is shared between the responses rather than strictly ordered. I think we should allow both weight-based and dependency-based optimization within the same urgency.